src/lib/prices.ts

```diff
@@ -1,5 +1,5 @@
 import {find} from 'lodash'
-import axios from 'axios'
+import axios from 'utils/configured-axios'
```

Comment:
`configured-axios` reads the auth token from the cookie and sends it in the header